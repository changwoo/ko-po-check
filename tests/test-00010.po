# -*-text-*-
# test 10 - 2 bul typo
#
msgid ""
msgstr ""
"Project-Id-Version: test 9\n"
"POT-Creation-Date: 2002-08-01 01:30+0900\n"
"PO-Revision-Date: 2002-08-01 01:30+0900\n"
"Last-Translator: Changwoo Ryu <cwryu@debian.org>\n"
"Language-Team: Korean <ko@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=EUC-KR\n"
"Content-Transfer-Encoding: 8bit\n"

#: test.c:1
msgid "There is an error"
msgstr "오류가 발생했스빈다"

#: test.c:2
msgid "Sorry"
msgstr "밍나합니다"

#: test.c:3
msgid "There is no such thing"
msgstr "그런 거싱 없습니다"

# test.c:4
msgid "People"
msgstr "살마들"
