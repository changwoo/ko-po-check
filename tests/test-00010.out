test-00010.po:17: "스빈다": 두벌식 오타로 보입니다
test-00010.po:21: "밍나합니다": 두벌식 오타로 보입니다
test-00010.po:25: "거싱": 두벌식 오타로 보입니다
test-00010.po:29: "살마": 두벌식 오타로 보입니다
